pkg/controller/daemonset/daemonset_controller.go

```diff
@@ -925,13 +925,7 @@ func (dsc *ReconcileDaemonSet) getNodesToDaemonPods(ds *appsv1alpha1.DaemonSet)
 				pod.Namespace, pod.Name, ds.Namespace, ds.Name)
 			continue
 		}
-		node := &corev1.Node{}
-		err = dsc.client.Get(context.TODO(), types.NamespacedName{Name: nodeName}, node)
-		if err == nil && CanNodeBeDeployed(node, ds) {
-			nodeToDaemonPods[nodeName] = append(nodeToDaemonPods[nodeName], pod)
-		} else {
-			klog.V(4).Infof("get node: %s failed", nodeName)
```

Comment:
should just work for "node not found" error, not all kind of errors

Reply:
update and add some comments on this.